### ISSUE TITLE
For user it shows 3/3 invites despite only one person invited in the list of the user and backend

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#21226]
User View:
![image](https://user-images.githubusercontent.com/25908592/113413573-46261380-93bb-11eb-9c1d-52135be2e016.png)

My View:
![image](https://user-images.githubusercontent.com/25908592/113413588-4c1bf480-93bb-11eb-9156-0e6ffb0c880b.png)

User ID:
193585

**Comment by D3nnis3n** [comment]
User said he resent the same single invite multiple times.

**Comment by denysaw** [comment]
@D3nnis3n I found the problem why counts were differ. Admin area was showing real invitations from `invitations` table found by needed user_id, instead of `user.invitations_count`, which is shown to user on `/account` page. Now I've changed and showing you both:
![image](https://user-images.githubusercontent.com/5720101/113456574-d9883480-940d-11eb-82d1-363a2a50bc3a.png)

I'll deduct it to 1 for the current user. I attempted to fix this invitation count in theory. So should be fine now. On Prod also.
